CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ The Technical Team co-leaders will, from time to time, designate and appoint sen
 
 ## How to contribute
 
-We use GitHub as our main development platform. GitHub features a "Pull Request" mechanism for proposing contributions to the schema from our community. A brief overview of this process:
+We use GitHub as our main development platform. If you just want to propose a change in English (as opposed to proposing exactly how the schema should be changed), you can add a new Issue describing it. If you want to propose specific changes to the schema, GitHub features a "Pull Request" mechanism for this. A brief overview of this process:
```

Comment:
This is confusing. Why English? Do you mean "a change to the guidelines"? But most importantly we probably want to wait and adjust it until we have the new guidelines online.

Reply:
Yes, given the international nature of our community, the use of "English" to mean "plain-language," rather than the actual language, is likely to cause more confusion.

Reply:
I just looked at about 10 open Issues for this repository, and they're all in English; likewise the titles of all 43 open Issues. Nonetheless, a good point! I think the best wording would be to propose a change "verbally"; how's that sound?

Reply:
I don't really understand what you want to say here -- 'verbally' generally means "face-to-face, by speech" which wouldn't really fit either...

Reply:
I don't feel "verbally" implies "in speech", but okay. How about "in words"? You suggested "plain-language", but that doesn't seem clear to me. (Hyphenating seems odd to me, but I don't think leaving it out would help.)

Strictly speaking, I suppose the generalization of "in English" is "in natural language", but that sounds excessively techie to me!

Reply:
Again, I don't know what you're trying to say. Are you trying to say that people shouldn't be afraid of posting ideas to the GitHub issues list, instead of well-thought-out proposals for a schema change?

Reply:
Not quite. Jeez, I thought this was a simple, obvious thing, but maybe I'm way off base here; if so, I apologize profusely for wasting your time and that of others! Anyway, the current CONTRIBUTING.md says nothing about adding Issues anything like the last five (at a minimum, but probably a lot more) open issues, including my own, #486. All of those contain informal natural-language descriptions of changes desired, and I think several (despite using informal language) _are_ "well-thought-out proposals for a schema change". But I think CONTRIBUTING.md implies that the only way to contribute is by a Pull Request containing the _exact changes_ to the schema! Actually, I'll bet the vast majority of issues are informal descriptions; maybe the CONTRIBUTING.md doesn't mention it because it's considered so obvious that it's okay to add issues like that? And/or because people think informal proposals don't qualify as "contributions"?